book_storage.rb

```diff
@@ -0,0 +1,32 @@
+require './book'
+
+class BookStorage
+  def initialize
+    @books = []
+  end
+
+  def list_books
+    @books.each { |book| puts book }
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+
+    print 'Author: '
+    author = gets.chomp
+
+    book = Book.new(title, author)
+    @books.push(book)
+
+    puts 'Book created successfully'
+  end
```

Comment:
- [ ] good job cleaning up the main.rb file 👍 in this method text is printed to the console information is gathered and the gathered information is processed, this is not according to the single responsibility rule, try to split this in smaller easier to manage methods. for reference have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid).

Reply:
Thank, you I've addressed this